lib/app.js

```diff
@@ -123,6 +123,11 @@ module.exports = (app, options) => {
       renderer.renderToString(context, (err, renderedHtml) => {
         let html = renderedHtml
 
+        // Unfortunately, both vue template renderer *and*
+        // vue-cli ModernModePlugin inject script tags into the html, so
+        // the former ones are now being removed again
+        html = html.replace(/<script src="[^"]+" defer><\/script>/g, '')
```

Comment:
Not cool. Is there any way to suppress the template renderer's scripts injection?
ref https://github.com/vuejs/vue/issues/9847